HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -210,13 +210,20 @@ def customiseForOffline(process):
 
     return process
 
-def customizeForRecoPixelVertexing(process):
+def customizeHLTfor41058(process):
```

Comment:
To address https://github.com/cms-sw/cmssw/pull/41058#discussion_r1139306442 (just fyi, @mmusich).